homeassistant/components/canary/__init__.py

```diff
@@ -1,9 +1,12 @@
 """Support for Canary devices."""
+from __future__ import annotations
+
 from datetime import timedelta
 import logging
+from typing import Final
 
 from canary.api import Api
-from requests import ConnectTimeout, HTTPError
+from requests.exceptions import ConnectTimeout, HTTPError
```

Comment:
What's the point?

Reply:
since `ConnectionTimeout` does not exists in `requests` itself, but `requests.exceptions` ... got before
`error: Module 'requests' has no attribute 'ConnectTimeout'  [attr-defined]`

Reply:
Ok, it seems `requests` doesn't handle re-exports properly.